packages/client/components/Dashboard.tsx

```diff
@@ -29,6 +30,33 @@ const NewTeam = lazy(() =>
   )
 )
 
+const getShowFAB = (location: NonNullable<RouteProps['location']>) => {
+  const {pathname} = location
+  return (
+    pathname.includes('/me/tasks') ||
```

Comment:
Not sure how to test if the trailing slash is present or not with less repetition

Reply:
don't use `strict == true`